### ISSUE TITLE
Feature request: masking phone input

### ISSUE THREAD

**Issue by cloudorbush** [garganurag893/react-native-phone-number-input#66]
Instead of a number looking like 

`+12353535353`

It would be nice if it would show up as

` +1 (555) 555-1234`
 
 while the user is typing in real time.
 
 Thank you for making an awesome component.

**Comment by Davete0302** [comment]
Do have a workaround for this?

**Comment by cloudorbush** [comment]
@Davete0302 use another library

**Comment by mariomurrent-softwaresolutions** [comment]
> @Davete0302 use another library

Which one would you suggest?

**Comment by e-itegbe** [comment]
@mariomurrent-softwaresolutions did you end up using another library? I'm trying to figure out what library to use besides this. I don't know if its still actively maintained?

**Comment by cloudorbush** [comment]
@e-itegbe

- https://github.com/CaioQuirinoMedeiros/react-native-mask-input
- https://github.com/akinncar/react-native-mask-text

This is just for the masking.

Unfortunately I don't think there's a good all-in-one solution available, but let me know if you find it.

**Comment by mariomurrent-softwaresolutions** [comment]
@e-itegbe no, I'm still using this one, but thinking to migrate to react-native-mask-input
